### PR TITLE
Updated: Time & Weather (based on #2395)

### PR DESCRIPTION
**Pull request by kaffolder7**
_This PR is based on @sudeepban's initial work in PR https://github.com/tidbyt/community/pull/2395._

## Changes / additions
- Adds additional API's _(e.g. OpenWeather, OpenWeather One Call API 3.0, Tomorrow.io, & Open-Meteo)_
- Removes AccuWeather API, as there was a legal issue documented by @rohansingh in _[this Tidbyt forum post](https://discuss.tidbyt.com/t/weather-app/5115/40)_
- Adds additional weather metrics and animates through them based on what is selected via app schema. _(Default is still set to displaying only 'wind speed'. More weather metrics may be enabled via schema toggles.)_

###### Initial app load state _(before API key is entered)_:
CleanShot 2024-05-14 at 23 23 16@2x

###### Preview when a single weather metric is enabled _(e.g. 'wind speed')_:
CleanShot 2024-05-14 at 23 29 51@2x

###### Preview when multiple weather metrics are enabled _(e.g. 'wind speed', 'humidity', 'dew point', 'cloud coverage', & 'pressure')_:
CleanShot 2024-05-14 at 23 32 29

### CODE REVIEW

**Comment by kaffolder7** [comment]
@sudeepban – What are your thoughts on merging this PR in w/ yours? I added additional weather API's (since we [cannot use AccuWeather](https://discuss.tidbyt.com/t/weather-app/5115/40)) and added additional functionality as described in the PR's description. Hoping you're ok with how I've expanded upon your initial PR to add to your Time & Temp app.

Also, was wondering if maybe it makes sense to rename this app to be _"Time & Weather"_ or _"Time & Current Weather"_, or something else to indicate that this app displays more than just the current time and temperature.

If you're comfortable with these changes, then we can try and get @matslina to approve. 🙂

**Comment by sudeepban** [comment]
@kaffolder7 that's very cool, nice work! I am definitely supportive of this and am glad to have you carry the torch and hopefully find a way to get this out with @matslina's approval. Appreciate the effort!

**Comment by sudeepban** [comment]
Also no strong feelings about the name of the app, agree that Time & Temp seems limiting and am fine with a change there.

**Comment by kaffolder7** [comment]
> Also no strong feelings about the name of the app, agree that Time & Temp seems limiting and am fine with a change there.

Thanks for your kind feedback @sudeepban and approval. Let's see if @matslina can get around to reviewing and hopefully accepting/merging this into `main` so more users can access this on their Tidbyts. :)

I also went ahead and changed the name of the app to better-reflect the functionality of this app.

File: apps/timeandweather/time_and_weather.star [hunk 1]
```diff
@@ -0,0 +1,1894 @@
...
+nLuarrlca7nuer21e2b36RueN87d9L158Rb/1tWeOT3dvfN6b/fF9/XfFt1+cif9
+zsu72Xcn7q28T7xf9EDtQdlD3YfVP1v+3Njv3H9qwHeg89HcR/cGhYPP/pH1jw9D
+BY+Zj8uGDYbrnjg+OTniP3L96fynQ89kzyaeF/6i/suuFxYvfvjV69fO0ZjRoZfy
+l5O/bXyl/erA6xmv28bCxh6+yXgzMV70VvvtwXfcdx3vo98PT+R8IH8o/2j5sfVT
+0Kf7kxmTk/8EA5jz/GMzLdsAAAAgY0hSTQAAeiUAAICDAAD5/wAAgOkAAHUwAADq
+YAAAOpgAABdvkl/FRgAAAEdJREFUeNrEj8ENACAIxMCN2H+IspE+iMaAxqd9Aekl
+nMgfdF+AGMzsmgD6BFiZq7TbbynZLVR3Pz6z7poaVKKT1rLJ+MsYAGKWVKuIhuN3
+AAAAAElFTkSuQmCC
+""",
+    "foggy.png": """
```
Comment (matslina) [inline apps/timeandweather/time_and_weather.star:102]: You can try running `exiftool -all=` on assets like these to trim down their size by removing EXIF data. On this particular one, it drops from 2.8k to 212 bytes.

**Review by matslina — APPROVED** [review]

**Pull request merged**